commands/alpha/rpkg/pull/command_test.go

```diff
@@ -68,13 +68,22 @@ data:
 apiVersion: config.kubernetes.io/v1
 kind: ResourceList
 items:
+- apiVersion: kpt.dev/v1
+  kind: KptRevisionMetaData
+  metadata:
+    name: bar
+    annotations:
+      internal.kpt.dev/resource-version: "999"
```

Comment:
If we have a separate file, it seems a bit strange to include this information in an annotation. Have we considered putting this in a dedicated field in the spec?

Reply:
As per the discussion, PackageRevisionResources metadata includes the resourceVersion and additional details. So will be dumping the content directly from there to this file.